APE_Bayesian_modeling.rst

```diff
@@ -143,46 +143,47 @@ without any prior, this function would just give a constant prior (e.g., 1).
 probability is reported as ln(prob).
 
-The current set of fitters are based on optimizers that minimize the statistic
-and the only currently implemented statistic is the Gaussian chisqr.  At a
-minimum, a new statistic that combines a Gaussian probability (i.e.,
-exp(-chisqr/2)) with the prior function is needed.  An alternative would be to
-create a new class (`Posterior`?) that combined both a statistic (i.e.,
-likelilhood) and prior giving the function to be maximized.
-
-Fitters
-=======
-
-A new class of fitters needs to be added.  This is the ``Sampler`` class that would
-provide the interface to different samplers.  Unlike the existing Optimizers
-where the best fitting model is the result, Samplers provide a sampling of the
-posterior where the number of samples is set as an input.  Effectively, the
-samplers provide a set of models and the set reflects the posterior. This set
-of models is usually returned as a 1D array of for each model parameter
-(called a chain) where the posterior is reflected as the frequency of a model
-parameter in the chain or as a parallel 1D array of posterior weights.
-The chain for each parameter will be stored in `Parameter.sampled_posterior` as an
-``astropy.uncertainty`` distribution.
+The current set of fitters are based on optimizers, which minimize the statistic.
+Currently, the only implemented statistic is the Gaussian chisqr.  At a minimum,
+a new statistic that combines a Gaussian probability (i.e., exp(-chisqr/2)) with
+the prior function is needed.  An alternative would be to create a new class
+(`Posterior`?) that combines both a statistic (i.e., likelilhood) and prior
+allowing the function to be maximized.
+
+Samplers
+========
+
+A new module akin to fitting needs to be added.  This module would be based
+around a ``Sampler`` class (similar to the ``Fitter`` class), which would
+provide the generic interface for different samplers.  Unlike the existing
+Optimizers where the best fitting model is the result, Samplers provide a
+sampling of the posterior where the number of samples is set as an input.
+Effectively, the samplers provide a set of models where the set reflects the
+posterior.  This set of models is usually returned as a 1D array of values
+for each model parameter (called a chain), where the posterior is reflected
+as the frequency of a model parameter in the chain or as a parallel 1D array
+of posterior weights. The chain for each parameter will be stored in
+`Parameter.sampled_posterior` as an ``astropy.uncertainty`` distribution.
 
 At least two samplers will be implemented.  The specific samplers planned are
 `emcee` and `dynasty`.
 
 The current ``Optimizer`` class should be abstracted to a generic class
-(``Solver``?) that provides the common portions of the generic ```Optimizer`` and
+(``Solver``?) that provides the common portions of generic ```Optimizer`` and
 ``Sampler`` classes.  The ``Optimizer`` and ``Sampler`` will provide a specific
```

Comment:
Can you change "Optimizer" to "Optimization"?  I see this is the actual class name in optmizers.py